### ISSUE TITLE
Welcome View - buttons should have aria-label set

### ISSUE THREAD

**Issue by kupatkar99** [microsoft/vscode#160396]
### GitHub Tags:
#A11yMAS; #A11yTCS; #A11ySev2; #DesktopApp; #Win32; #Visual Studio Code Client; #BM-VisualStudioCodeClient-Win32-Aug2022; #WCAG4.1.2; #Name Role Value

### Environment and OS details:
Visual Studio Code
Windows 11 Pro version 21H2 (OS Build 22000.856)

### Repro Steps:
1)Open Visual studio code editor
2)Using "Alt+H" key navigate to the "Help" menu bar option .
3)Using "Down" arrow key navigate to the "Welcome" item.
4) Make sure you have the following setting in settings.json: "workbench.startupEditor": "gettingStarted"
5)Using "Alt+H" navigate to "Help" menu bar option and use "down" arrow key to navigate to "Getting started" and observe that the correct name is defined for the "cancel" or "hide" button.

### Actual Result:
The Correct name is not defined for the "cancel" or "hide" button.

### Expected Result:
The correct name should be defined for the "cancel" or "hide" button.

![A11y_Visual Studio Code Client_Welcome Screen_Name Role Value (2)](https://user-images.githubusercontent.com/98530069/189107426-e0343881-d822-4762-bf0a-5b155d0ce1ee.png)

**Comment by isidorn** [comment]
For me the correct title gets read out by VoiceOver or NVDA - it is Hide. 

@lramos15 can you just make sure to also set the `aria-label` on the element? This is affecting our Accessibility Grade and I think it is a very low hanging fix - thus assigning to November for now.

**Comment by lramos15** [comment]
@isidorn @sbatten has planned the accessibility work for the welcome view for the new team member. I'll ask them to verify this fix 👍
